sashimi/processes/camera.py

```diff
@@ -123,13 +123,18 @@ def __init__(
         self.was_waiting = False
 
     def initialize_camera(self):
-        if conf["scopeless"]:
-            self.camera = camera_class_dict["mock"]()
-        else:
-            self.camera = camera_class_dict[conf["camera"]["name"]](
-                camera_id=conf["camera"]["id"],
-                max_sensor_resolution=tuple(conf["camera"]["max_sensor_resolution"]),
-            )
+        camera_conf = conf.pop("camera")
```

Comment:
Why are you popping the keys from the dictionary instead of just accessing them?

Reply:
I like the idea of configuration parameters being consumed only once. @vilim ?